WordPressAuthenticator/Signin/LoginLinkRequestViewController.swift

```diff
@@ -24,6 +24,10 @@ class LoginLinkRequestViewController: LoginViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        // TODO: remove before merging
+        navigationItem.title = WordPressAuthenticator.shared.displayStrings.logInTitle
```

Comment:
What happens to the large title if an app puts in a blank string or nil?

Reply:
If large titles are enabled but no title is set, then the large title area is displayed with no title:

<img width="400" alt="no_title" src="https://user-images.githubusercontent.com/1816888/85165816-a03a5b80-b223-11ea-9be9-639b4b2ec790.png">